components/automate-load-balancer/habitat/plan.sh

```diff
@@ -1,16 +1,37 @@
 pkg_name=automate-load-balancer
-pkg_description="internal and external load balancer and reverse proxy for Automate 2.0"
 pkg_origin=chef
 pkg_version="0.1.0"
+pkg_description="internal and external load balancer and reverse proxy for Automate 2.0"
```

Comment:
Most of the changes here were copied from here https://github.com/chef-base-plans/nginx/blob/master/plan.sh